cpp/src/pypowsybl.cpp

```diff
@@ -1093,6 +1093,13 @@ std::vector<SeriesMetadata> getTwtFeederBaysMetadata() {
     return res;
 }
 
+SeriesArray* getConnectablesOrderPositions(const JavaHandle& network, const std::string voltage_level_id) {
+    return new SeriesArray(callJava<array*>(::getConnectablesOrderPositions, network, (char*) voltage_level_id.c_str()));
+}
 
+std::vector<int> getUnusedConnectableOrderPositions(const pypowsybl::JavaHandle network, const std::string busbarSectionId, const std::string beforeOrAfter) {
+    auto positionsArrayPtr = callJava<array*>(::getUnusedConnectableOrderPositions, network, (char*) busbarSectionId.c_str(), (char*) beforeOrAfter.c_str());
+    return toVector<int>(positionsArrayPtr);
```

Comment:
I think there is a memory leak: we need to free the `array*` returned from java here
(I see there is another one in the existing code!)